src/veritysetup/veritysetup.c

```diff
@@ -196,6 +200,21 @@ static int parse_options(const char *options) {
                         r = free_and_strdup(&arg_hash, val);
                         if (r < 0)
                                 return log_oom();
+                } else if ((val = startswith(word, "fec-device="))) {
+
+                        r = free_and_strdup(&arg_fec_what, val);
+                        if (r < 0)
+                                return log_oom();
+                } else if ((val = startswith(word, "fec-offset="))) {
+
+                        r = safe_atou64(val, &arg_fec_offset);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to parse %s, ignoring: %m", word);
+                } else if ((val = startswith(word, "fec-roots="))) {
+
+                        r = safe_atou64(val, &arg_fec_roots);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to parse %s, ignoring: %m", word);
```

Comment:
check range here? the man page says 2…24 is ok, maybe enforce this here and log if outside of range

Reply:
done.